src/classes/GE_PaymentServices.cls

```diff
@@ -45,7 +45,8 @@ public with sharing class GE_PaymentServices {
         DECLINED,
         NONRETRYABLEERROR,
         RETRYABLEERROR,
-        REFUNDISSUED
+        REFUNDISSUED,
+        SUBMITTED
```

Comment:
There is also a SETTLED status isn't there?

Reply:
@KietTruongSFDO I'm actually not going to add anything for these now since I don't need it for the logic.